clang/include/clang/CIR/Dialect/IR/CIROps.td

```diff
@@ -1447,13 +1447,13 @@ def VTableAddrPointOp : CIR_Op<"vtable.address_point",
 
 def SetBitfieldOp : CIR_Op<"set_bitfield"> {
   let summary = "Set a bitfield";
-  let description = [{ 
-    The `cir.set_bitfield` operation provides a store-like access to 
+  let description = [{
+    The `cir.set_bitfield` operation provides a store-like access to
     a bit field of a record.
 
     It expects an address of a storage where to store, a type of the storage,
     a value being stored, a name of a bit field, a pointer to the storage in the
-    base record, a size of the storage, a size the bit field, an offset 
+    base record, a size of the storage, a size the bit field, an offset
```

Comment:
Please leave these format changes out as they just generate noise for reviewers (you can set up your IDE to only format code snippets that actually change) - I'm gonna soon format all of CIR to remove these and turn on clang-format-action (see #345)

Reply:
These aren't really formatting changes. They only remove trailing whitespaces. I'd rather avoid turning that off again, since it'll just result in me doing more work while contributing to other projects. I'd be happy to make a sweep of the CIR sources to remove all the trailing whitespaces in an NFC patch.

Reply:
I didn't mean your whitespace changes were intentional formatting. They usually come from an IDE setup to format the entire file when you click save, which can easily be changed to only format the lines of code you change when saving. Unfortunately removing trailing whitespaces do generate some unnecessary noise - this is not clangir specific, the same rule will apply to any file in llvm-project.

I should have mentioned that in your last PR, so you wouldn't need to go to the trouble of doing it in this one - your previous one had the same problem, which I intentionally overlooked since there weren't that many... my bad! So this one is one me.

> I'd be happy to make a sweep of the CIR sources to remove all the trailing whitespaces in an NFC patch.

That'd be great, thanks! Any chance you can go the extra length and enable #345 as part of such PR?

Reply:
It's not a problem in libc++ or clang, since I've also removed trailing whitespaces there. I'll look into adding formatting tomorrow. That shouldn't be super difficult.